test/tests/jobs/UpdateJobTest.cpp

```diff
@@ -0,0 +1,53 @@
+#include <test/lib/BedrockTester.h>
+#include <test/tests/jobs/JobTestHelper.h>
+
+struct UpdateJobTest : tpunit::TestFixture {
+    UpdateJobTest()
+            : tpunit::TestFixture("UpdateJob",
+                                  BEFORE_CLASS(UpdateJobTest::setupClass),
+                                  TEST(UpdateJobTest::updateJob),
+                                  AFTER(UpdateJobTest::tearDown),
+                                  AFTER_CLASS(UpdateJobTest::tearDownClass)) { }
+
+    BedrockTester* tester;
+
+    void setupClass() { tester = new BedrockTester(_threadID, {{"-plugins", "Jobs,DB"}}, {});}
+
+    // Reset the jobs table
+    void tearDown() {
```

Comment:
you shouldn't need this, every tester gets it's own db, etc. so when this tested is deleted in "tearDownClass()` all of the data is deleted.

Reply:
I imagine you added it in case there are more tests added? but yeah, don't add it if you're not adding more tests. (tearDown is between function calls here @coleaeason , not between classes)

Make sure you remove it from line 9 as well.

Reply:
>(tearDown is between function calls here @coleaeason , not between classes)

this makes a lot more sense